cookie clicker

FisherXZ opened the pull request:
calculate click speed
show a progress bar of how fast you're clicking

background color change every 20 click

sampoder committed:
Initial commit

sampoder committed:
Update README.md

sampoder committed:
Update README.md

sampoder committed:
Update README.md

sampoder committed:
Update README.md

sampoder committed:
Update README.md

sampoder committed:
Update README.md

sampoder committed:
Update README.md

sampoder committed:
Update README.md

sampoder committed:
Create README.txt

sampoder committed:
Update README.txt

FisherXZ committed:
Created my website and a earth website

FisherXZ committed:
Delete .DS_Store

FisherXZ committed:
Delete week1/.DS_Store

FisherXZ committed:
new

FisherXZ committed:
Merge branch 'main' of https://github.com/fischerxz/custartspring25

FisherXZ committed:
made my personal website so better

FisherXZ committed:
faster rotation

FisherXZ committed:
cookie clicker

etam1 reviewed (changes requested):
hello please have your work under the format "week3/firstname_lastname" (with not space in between week and 2

FisherXZ committed:
week 4

made a lot of crazy fascinating changes to the game

sorry submitted late

please try game, it's very fun

FisherXZ committed:
week 5

FisherXZ committed:
chat + mongodb

FisherXZ committed:
improved chatbot

- added rooms and users
- tried adding a genai chat bot but failed

FisherXZ committed:
ss
